modules/scheduling-a-pipeline-run.adoc

```diff
@@ -33,7 +33,7 @@ The *Create run* page opens.
 * Select *Run once immediately after creation* to specify the run executes once, and immediately after its creation.
 * Select *Schedule recurring run* to schedule the run to recur.
 ... Configure the run's trigger type.
-.... Select *Periodic* and select the execution frequency from the list.
+.... Select *Periodic* to specify an execution frequency. Enter a numerical value and select an execution frequency from the list.
```

Comment:
please clarify - line 36 - should you changed **and** to **or**? 
That is, the either type in a value of select one from the list (they don't do both, right?)

Reply:
It's both. They need to type in a value and select an appropriate frequency from the list.

For example, entering 10 and selecting hours from the list executes a pipeline run every 10 hours.

Reply:
thanks for explaining - I think it will be obvious to the user when looking at the UI